eng/Signing.props

```diff
@@ -7,12 +7,9 @@
   </ItemGroup>
   <ItemGroup Condition="'$(PostBuildSign)' != 'true'">
     <ItemsToSign Remove="@(ItemsToSign)" />
-    <ItemsToSign Include="$(ArtifactsShippingPackagesDir)*.zip" Condition=" '$(PublishBinariesAndBadge)' == 'true' " />
-    <ItemsToSign Include="$(ArtifactsShippingPackagesDir)*.exe" />
-    <ItemsToSign Include="$(ArtifactsShippingPackagesDir)*.msi" />
-    <ItemsToSign Include="$(ArtifactsNonShippingPackagesDir)*.msi" />
-    <ItemsToSign Include="$(ArtifactsNonShippingPackagesDir)*.zip" Condition=" '$(PublishBinariesAndBadge)' == 'true' " />
-    <ItemsToSign Include="$(ArtifactsPackagesDir)**\*.wixpack.zip" />
+    <ItemsToSign Include="$(ArtifactsPackagesDir)**\*.exe" />
```

Comment:
Just also noticed the `$(ArtifactsNonShippingPackagesDir)*.exe` files were specifically excluded before. Doesn't hurt much to sign them (if any exist) of course.

Reply:
They weren't excluded. There just weren't any so they weren't added.